cla_common/constants.py

```diff
@@ -515,8 +515,7 @@
 )
 
 OPERATOR_HOURS = {
-    "weekday": (datetime.time(9, 0), datetime.time(20, 0)),
-    "saturday": (datetime.time(9, 0), datetime.time(12, 30)),
+    "weekday": (datetime.time(9, 0), datetime.time(17, 0)),
     "2018-12-24": (datetime.time(9, 0), datetime.time(17, 30)),
     "2018-12-31": (datetime.time(9, 0), datetime.time(17, 30)),
     "2019-12-24": (datetime.time(9, 0), datetime.time(18, 00)),
```

Comment:
Is this tech debt that we could expunge now?

Reply:
It could almost certainly go. I'm not sure I see much point in doing so, but we could